openedx/MITx-edx-integration-tutor.md

```diff
@@ -0,0 +1,199 @@
+---
+parent: OpenedX
+---
+# Integrating MIT Applications with Open edX using Tutor
+
+| Application | PORT | Domain               |
+|-------------|------|----------------------|
+| MITxPro     | 8053 | xpro.odl.local       |
+| MITxOnline  | 8013 | mitxonline.odl.local |
+
+In order to create user accounts in Open edX and permit authentication from MIT Application to Open edX, you need to configure MIT Application as an OAuth2 provider for Open edX.
+
+    These instructions should work for a Tutor Dev or Tutor Nightly deployment as well. Specify `--tutor-dev` instead of `--tutor` when running `configure_instance` so the URLs have a port on them.
+
+## Preliminary Step
+
+`pyenv` (and `pyenv-virtualenv`) are highly recommended for managing local Python versions. `Use the instructions on their [GitHub page](https://github.com/pyenv/pyenv) to install if you haven't already installed it.`
+
+You'll want to create at least a virtualenv for Tutor. As of this writing, Tutor uses Python 3.8.12 (in the LMS container at least); I have also successfully used 3.9.16. 3.11 has *not* worked for me.
+
+## Tutor Setup, Part One
+
+    Note that no hosts file changes are needed if you use the default `local.edly.io` domain - that's a real domain with a wildcard subdomain cname that points to 127.0.0.1.
+
+To begin, you need to follow the `[One-Click Installer](https://docs.tutor.overhang.io/quickstart.html)` instructions provided by Tutor. Do this with your Tutor virtualenv activated.
```

Comment:
We should probably note that the instructions here are for a dev instance (`tutor dev` rather than `tutor local`). The dev and local setups work differently - I think it'd be better if we standardized on the dev one (since it's likely we'll need to actually do work on edX).